src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,33 @@
 using namespace std;
 
 int t06_cinema() {
-    
+    int n,m,k,counter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'counter' is not assigned a value.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4568488)